monkey/monkey_island/cc/utils/field_encryptors/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .i_field_encryptor import IFieldEncryptor
+from .mimikatz_results_encryptor import MimikatzResultsEncryptor
+from .string_list_encryptor import StringListEncryptor
```

Comment:
Should we consolidate this with the rest of the encryption code under `cc/server_utils/encryption`?